### PR TITLE
AI Tutor: UI update - tutorTypeSelector rather than radio button toggles 

### PR DESCRIPTION
**Pull request by Erin007**
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx [hunk 1]
```diff
@@ -50,15 +55,9 @@ const AITutorPanel: React.FunctionComponent<AITutorPanelProps> = ({open}) => {
     },
   ];
 
-  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
-    e.preventDefault();
-    dispatch(addAIResponse(''));
-    setSelected(e.target.value);
-  };
-
```
Comment (kakiha11) [inline apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx:54]: @Erin007 I wasn't 100% sure about removing the dispatch line but I figured we could probably move that functionality somewhere else if it's still needed now that the onChange doesn't apply

**Comment by kakiha11** [comment]
@Erin007 here is where I got the panel to with these updates! 

A coding level with validation

aiTutorAllThree

A coding level without validation

aiTutorOnlyTwo

Turns out there aren't any non-coding non-assessment levels in CSA that I could find so hard to demo the just "General Chat " possibility.

File: apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx (hunk 1 above)
Comment (Erin007) [reply apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx:54]: I think that's totally fine. The dispatch line was just setting the AI's response to an empty string, which you're right, we'll end up doing somewhere else if we need it.

**Review by Erin007 — COMMENTED** [review]
I think this is a big step in the right direction and we can continue to iterate from here. Nice job, Ken! Github won't let me approve the PR because I opened it, but 👍 🚀 ✅

**Review by kakiha11 — APPROVED** [review]

**Pull request merged**